api/db/queries.js

```diff
@@ -1,8 +1,15 @@
 const { Pool } = require('pg');
+require('dotenv').config();
 
 // pools will use environment variables
 // for connection information
-const pool = new Pool();
+const pool = new Pool({
+  user: process.env.DB_USER,
+  host: process.env.DB_HOST,
+  database: process.env.DB_DATA,
+  password: process.env.DB_PASS,
+  port: process.env.DB_PORT,
+});
```

Comment:
I would consider making this three separate files. One to handle the db connection, one to control question queries, and one to control answer queries.